requirements.txt

```diff
@@ -60,26 +73,35 @@ prison==0.1.2             # via flask-appbuilder
 py==1.8.1                 # via retry
 pyarrow==0.15.1
 pycparser==2.19           # via cffi
-pyjwt==1.7.1              # via flask-appbuilder, flask-jwt-extended
+pycryptodomex==3.9.6      # via snowflake-connector-python
+pyjwt==1.7.1              # via flask-appbuilder, flask-jwt-extended, snowflake-connector-python
+pyopenssl==19.1.0         # via snowflake-connector-python
 pyrsistent==0.15.7        # via jsonschema
 python-dateutil==2.8.1
 python-dotenv==0.10.5
 python-editor==1.0.4      # via alembic
 python-geohash==0.8.5
 python3-openid==3.1.0     # via flask-openid
-pytz==2019.3              # via babel, celery, pandas
+pytz==2019.3              # via babel, celery, pandas, snowflake-connector-python
 pyyaml==5.3
+requests==2.22.0          # via azure-storage-common, snowflake-connector-python
 retry==0.9.2
+s3transfer==0.3.2         # via boto3
 selenium==3.141.0
 simplejson==3.17.0
-six==1.14.0               # via bleach, cryptography, flask-jwt-extended, flask-talisman, isodate, jsonschema, pathlib2, polyline, prison, pyarrow, pyrsistent, python-dateutil, sqlalchemy-utils, wtforms-json
+six==1.14.0               # via bleach, cryptography, flask-jwt-extended, flask-talisman, isodate, jsonschema, pathlib2, polyline, prison, pyarrow, pyopenssl, pyrsistent, python-dateutil, sqlalchemy-utils, wtforms-json
+snowflake-connector-python==2.2.0  # via snowflake-sqlalchemy
+snowflake-sqlalchemy==1.2.0
```

Comment:
`extra_requires` should not be included in `requirements.txt`. This change is also pulling in libraries that are not apache compatible, at least `requests` but perhaps others, too.